### DISCUSSION TITLE
[Bug]: Angular ɵɵinvalidFactoryDep after upgrading to storybook 7

### DISCUSSION THREAD

**Discussion by sagnikpalchowdhury** [storybookjs/storybook#25995] [Help] [answered]
### Summary

Hi, I was hoping to know how everyone solved/ found a work around for this issue https://github.com/storybookjs/storybook/issues/23534 ?

 I am upgrading Storybook from v6 to v7. 

My storybook version is 7.4.2 and am using Angular  version 15.2.10.

**Comment by Marklb** [comment, marked as answer]
As far as I know, the workaround is already mentioned in the comments. https://github.com/storybookjs/storybook/issues/23534#issuecomment-1665244348

Are you able to provide a reproduction?

I don't remember if a reason was ever figured out, but the issue is still tagged as needing a reproduction, which would help find the problem much faster or at least be able to hopefully explain the cause.

If you are unable to provide a reproduction, would you be able to try the v8 beta and see if any other changes may have fixed it?

**Comment by valentinpalkovic** [comment]
Duplicate of https://github.com/storybookjs/storybook/issues/23534. Please continue the discussion there.
